content/en/docs/Installation/trustedboot.md

```diff
@@ -121,6 +121,43 @@ title My Awesome OS
 Remember when you build upgrade mediums, to also add your branding, otherwise the new active image will be named differently than your fallback and recovery ones.
 {{% /alert %}}
 
+### Additional efi entries
```

Comment:
to not confuse the user here, I'd move this down to Reference/Note.

I'd keep on the first part just the "minimum to follow", this looks like additional/advanced usage that a user might need after taking confidence with the process

Reply:
You mean I should move this whole section? Which part should I keep here?

Reply:
Done, moved